core/iwasm/interpreter/wasm_loader.c

```diff
@@ -2920,6 +2904,29 @@ load_memory_import(const uint8 **p_buf, const uint8 *buf_end,
         declare_init_page_count = spectest_memory_init_page;
         declare_max_page_count = spectest_memory_max_page;
     }
+#if WASM_ENABLE_LIB_WASI_THREADS != 0
+    /* a case in wasi-testsuite which imports ("foo" "bar") */
```

Comment:
if so(to make test cases happy), we'd better make a new preprocessor conditionals. like `WASM_ENABLE_WASI_TEST`.

@wenyongh we need a new marco here to control if in some wasi-testsuites